### PR TITLE
(DEV-3892) Adds "With You" Screen After Adding Prayer

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

This adds a screen that let's them go straight to praying for other people or go back and add another prayer.

### How do I test this PR?

Add some prayers!

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/newspringchurchapp/src/prayer/AddPrayer/WithYou/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+const FooterAltOption = styled({
+  alignSelf: 'center',
+})(PaddedView);
+
+const FooterText = styled(({ theme }) => ({
+  color: theme.colors.tertiary,
+}))(BodyText);
+
+const WithYou = memo(({ ...props }) => (
+  // TODO: this should clear out the stack instead of just
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/AddPrayer/WithYou/index.js]:
```suggestion
```

File: packages/newspringchurchapp/src/prayer/AddPrayer/WithYou/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  // navigating, leaving the AddPrayerForm and this screen in the stack
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/AddPrayer/WithYou/index.js]:
```suggestion
```

File: packages/newspringchurchapp/src/prayer/AddPrayer/WithYou/index.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        <Title>The NewSpring family is with you in prayer</Title>
+        <BodyText>
+          Thank you for sharing your prayer request. We will all be praying for
+          you.
+        </BodyText>
+      </MessageView>
+      <Footer>
+        <Button
+          title="Pray for Others"
+          // TODO: this should pop off this and the AddPrayerForm first
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/AddPrayer/WithYou/index.js]:
```suggestion
```

**Review by richarddubay — CHANGES_REQUESTED** [review]
Currently the prayer list doesn't work ... but the screen looks great! Just get that prayer list working and I'll re-review.

File: packages/newspringchurchapp/src/prayer/PrayerList/index.js [hunk 4]
```diff
@@ -114,7 +114,7 @@ class PrayerList extends PureComponent {
     const { queryTag, label, variables } = this.calculateQuery();
 
     return (
-      <ModalView onClose={() => this.props.navigation.pop()}>
+      <ModalView onClose={() => this.props.navigation.navigate('Prayer')}>
```
Comment (richarddubay) [inline packages/newspringchurchapp/src/prayer/PrayerList/index.js]:
```suggestion
      <ModalView onClose={() => this.props.navigation.popToTop()}>
```

What do you think about this?

**Review by richarddubay — APPROVED** [review]
:shipit:

**Pull request merged**